src/Account.ts

```diff
@@ -202,14 +201,15 @@ export class Account {
 
   /**
    *
-   * @param params An object with key amount signifying the amount of debt tokens to return and key vault indicating the Contract
+   * @param params An object with key amount signifying the amount of debt tokens
+   *  to return, key vault indicating the Contract and key close indicating if the vault should be closed
    * @returns A boolean indicating if the vault debt was returned or not
    */
-  async returnVaultDebt(params: { amount: number; vault: Vault }): Promise<boolean> {
+  async returnVaultDebt(params: { amount: number; vault: Vault; close: boolean }): Promise<boolean> {
```

Comment:
Is it possible to set the default value for this to be `false`?

Reply:
@AwolDes 
Done